src/gui/abstractfileitemwidget.cpp

```diff
@@ -185,8 +185,7 @@ void AbstractFileItemWidget::setPath(const QString &path) {
     _driveIconLabel->setPixmap(
         KDC::GuiUtility::getIconWithColor(":/client/resources/icons/actions/icon-folder-empty.svg").pixmap(iconSize));
 
-    QFileInfo fileInfo(path);
-    QString printablePath = fileInfo.dir().path();
+    QString printablePath = "/" + path;
```

Comment:
This pattern is quite common in the rest of the code. Maybe for later: write a helper that instantiates the path to be displayed with a single call.

Reply:
Yes agreed. The path are often poorly managed.
In this specific case, this is also to make it more clear that it is a link and clicking on it will open the destination folder

Reply:
If ok with you I will leave it like this for now. Paths will need a deeper general refactoring some day